specification/configuration.md

```diff
@@ -133,7 +133,10 @@ beyond the OpenTelemetry specification exist.
 - Zipkin exporter
   - Distribution MUST NOT list Zipkin exporter as supported (not supported by Smart Agent)
 - `OTEL_TRACES_EXPORTER`
-  - Non-RUM distribution MUST default to `otlp` over gRPC with an endpoint of `localhost:4317`
+  - Non-RUM distribution MUST support following, additional values:
```

Comment:
I think that `MUST` in this context introduced a breaking-change. Maybe it would be to change it to `SHOULD`? Otherwise, I think we should bump the major version of the spec.

Reply:
I prefer MUST here, as SHOULD won't introduce a consistency between libraries. Then the docs will be much more messy and overall APM harder to use. Say:
- in python set single variable 
- in java set two, with different values
If we want to make the experience better for end-users, we need to have it supported everywhere.